pinot-query-runtime/src/main/java/org/apache/pinot/query/service/QueryConfig.java

```diff
@@ -24,8 +24,8 @@
 public class QueryConfig {
   public static final long DEFAULT_TIMEOUT_NANO = 10_000_000_000L;
 
-  public static final String KEY_OF_MAX_INBOUND_QUERY_DATA_BLOCK_BYTES_SIZE = "pinot.query.runner.max.msg.size";
-  public static final int DEFAULT_MAX_INBOUND_QUERY_DATA_BLOCK_BYTES_SIZE = 128 * 1024 * 1024;
+  public static final String KEY_OF_MAX_INBOUND_QUERY_DATA_BLOCK_SIZE_BYTES = "pinot.query.runner.max.msg.size.bytes";
```

Comment:
I guess this config hasn't been in use yet, right?

Reply:
Also, since this PR doesn't only modify the integration test, can we add some comments to the description section about this config change?

Reply:
this was a temporary fix, no longer needed to be this large after https://github.com/apache/pinot/pull/9485